src/grpc_stream_h.erl

```diff
@@ -325,21 +325,14 @@ send_request_body(Pid, Ref, fin, BodyLen, Data) ->
 %% @todo Better spec.
 -spec request_process(cowboy_req:req(), cowboy_middleware:env(), [module()]) -> ok.
 request_process(Req, Env, Middlewares) ->
-	OTP = erlang:system_info(otp_release),
 	try
 		execute(Req, Env, Middlewares)
 	catch
-		exit:Reason ->
-			Stacktrace = erlang:get_stacktrace(),
-			erlang:raise(exit, {Reason, Stacktrace}, Stacktrace);
-		%% OTP 19 does not propagate any exception stacktraces,
-		%% we therefore add it for every class of exception.
-		_:Reason when OTP =:= "19" ->
-			Stacktrace = erlang:get_stacktrace(),
+		exit:Reason:Stacktrace ->
 			erlang:raise(exit, {Reason, Stacktrace}, Stacktrace);
 		%% @todo I don't think this clause is necessary.
-		Class:Reason ->
-			erlang:raise(Class, Reason, erlang:get_stacktrace())
+		Class:Reason:Stacktrace ->
```

Comment:
I'm just not sure about this. Is the `Stacktrace` guarantied to be there?

Reply:
Yes, see https://www.erlang.org/doc/reference_manual/expressions.html#try
The old way (pre-OTP 21) was to use `get_stacktrace/0`.

Reply:
Great. OTP-21 is old enough that I think removing it is ok.